### PR TITLE
Adds some useful VS extensions

### PR DESCRIPTION
**Pull request by SteveALee**

### CODE REVIEW

File: windows-environment.md [hunk 1]
```diff
@@ -22,7 +22,12 @@ One of the pain points we hear from users is that the command line console in Wi
...
 
-* **[Node.js Tools for Visual Studio](https://aka.ms/explorentvs)** is a free, open-source extension that turns Visual Studio into a powerful Node.js IDE: intelligent code completions, advanced debugging and profiling, cloud deployment, unit-testing, REPL window, and more. For more information, check out this [overview video.](https://channel9.msdn.com/Blogs/Seth-Juarez/Nodejs-Tools-for-Visual-Studio)
+* **[Node.js Tools for Visual Studio](https://aka.ms/explorentvs)** is a free, open-source extension that turns Visual Studio into a powerful Node.js IDE: intelligent code completions, advanced debugging and profiling, cloud deployment, unit-testing, REPL window, and more. For more information, check out this [overview video.](https://channel9.msdn.com/Blogs/Seth-Juarez/Nodejs-Tools-for-Visual-Studio) Note that while VisualStudio can be used as a stand-alone additional IDE 'layer' to a comprehensive commandline based workflow, several VisualStudio extensions may also prove useful:
+  * [EditoConfig](https://visualstudiogallery.msdn.microsoft.com/c8bccfe2-650c-4b42-bc5c-845e21f96328) consistent editor spaces and indents.
```
Comment (mousetraps) [inline windows-environment.md]: *EditorConfig

Comment (mousetraps) [inline windows-environment.md]: Also Visual Studio, not "VisualStudio"

**Comment by mousetraps** [comment]
Minor comments and lgtm after that!

**Comment by mousetraps** [comment]
(also mind resolving the merge conflicts too? that way we can pull it in more quickly :smiley:)

**Comment by SteveALee** [comment]
@mousetraps Should be OK now

Is Task Runner Explorer now part of VS 2015? It's used by the npm task explorer and listed in the [nodesjtools home page](https://www.visualstudio.com/features/node-js-vs). I have it but not listed in my extensions.

Package Intellisense is also listed on the nodejstools page but not listed as available in VS2015 community.

**Comment by mousetraps** [comment]
:+1: 

Yeah, both task runner explorer and package intellisense are part of VS 2015 now, so we should update the docs accordingly.

**Pull request merged**